source/lex.tex

```diff
@@ -1136,11 +1136,11 @@
 \indextext{prefix!\idxcode{u8}}%
 is a character literal of type \tcode{char},
 known as a \defn{UTF-8 character literal}.
-The value of a UTF-8 character literal
+The value of a UTF-8 character literal containing a single \grammarterm{c-char}
 is equal to its ISO 10646 code point value,
 provided that the code point value
-is representable with a single UTF-8 code unit
-(that is, provided it is in the C0 Controls and Basic Latin Unicode block).
+is representable with a single UTF-8 code unit.
+(That is, provided it is in the C0 Controls and Basic Latin Unicode block.)
```

Comment:
I would prefer to change the `char16_t` wording to incorporate the parenthetical in the prior sentence rather than changing this wording to split it into a separate sentence.

Reply:
Yes, please amend and rebase this PR.